modules/manage/pages/kubernetes/troubleshooting/k-troubleshoot.adoc

```diff
@@ -359,8 +359,7 @@ NOTE: Kubernetes retains logs only for the current and the previous instance of
 
 . Reset the crash counter to zero to allow Redpanda to restart. You can do any of the following to reset the counter:
 +
-- Update the redpanda.yaml configuration file. You can make changes to any of the following sections in the Redpanda Helm chart to trigger an update:
-* `config.cluster`
+- Make changes to any of the following sections in the Redpanda Helm chart to trigger an update:
```

Comment:
Could we add an example just to give users something to copy paste? Changing `crash_loop_limit` to any other integer would be a great option IMO as it won't really impact anything else in the cluster.

Reply:
Added an example in YAML